compiler/modules/nback/build.bal

```diff
@@ -225,6 +225,7 @@ function buildModule(bir:Module mod, llvm:Context context) returns llvm:Module|B
         llFuncTypes.push(ty);
         bir:InternalSymbol symbol = defn.symbol;
         llvm:FunctionDefn llFunc = llMod.addFunctionDefn(mangleInternalSymbol(modId, symbol), ty);
+        llFunc.setGC(options.gcName);
```

Comment:
I prefer to only call this only if options.gcName is not nil

Reply:
Fixed